### PR TITLE
use cwd instead of os's temp dir for creating artifacts

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
Signed-off-by: Gregorio Di Stefano <greg.distefano+github@gmail.com>

### CODE REVIEW

File: artifacts.go [hunk 1]
```diff
@@ -791,7 +818,7 @@ func getCandidatesForModify(path string, key []byte) ([]partition, bool, error)
 func modifyArtifact(c *cli.Context) error {
 	if c.NArg() == 0 {
 		return cli.NewExitError("Nothing specified, nothing will be modified. \n"+
-			"Maybe you wanted to say 'artifacts read <pathspec>'?", 1)
+			"Maybe you wanted to say 'oartifacts read <pathspec>'?", 1)
```
Comment (kacf) [inline artifacts.go]: Did you *really* want to say "oartifacts"? :-)

File: artifacts.go [hunk 2]
```diff
@@ -394,10 +400,6 @@ func signExisting(c *cli.Context) error {
 		return cli.NewExitError("Unsupported version of artifact file: "+string(ver), 1)
 	}
 
-	if err = tFile.Close(); err != nil {
```
Comment (GregorioDiStefano) [inline artifacts.go]: Do we need this? The closing of the fh is already deferred AND it reduces the cyclomatic complexity to 15 :)

Comment (kacf) [reply artifacts.go]: Yes, we need it because the file is later renamed, and this happens before the deferred call.

**Comment by GregorioDiStefano** [comment]
@

**Pull request closed without merging**